recipes/python-omero/build.sh

```diff
@@ -0,0 +1,4 @@
+cd lib/python
+
+CONDA_PYTHON=$(conda info --root)/bin/python
```

Comment:
Why is this so complicated?
Does a simple `${RECIPE_DIR}/setup.py install` not work?